src/move_basic.cpp

```diff
@@ -487,9 +492,10 @@ void MoveBasic::drawLine(double x0, double y0, double x1, double y1)
 
 void MoveBasic::run()
 {
-    ros::Rate r(20);
+    ros::Rate r(10);
```

Comment:
Why decrease the rate here?

Reply:
It was for debugging and should not be in the PR